docs/tutorials/tutorial-hdfs-logs-distributed-search-aws-s3.md

```diff
@@ -27,144 +27,201 @@ Before using Quickwit with an object storage, check out our [advice](../administ
 
-## Start your servers
+In AWS, you can create a security group to group these inbound rules. Check out the [network section](configure-aws-env.md) of our AWS setup guide.
+
+To make things easier, let's create a security group that opens the TCP/UDP port range [7100-7400]. Next, create three EC2 instances using the previously created security group. Take note of each instance's public IP address. 
 
-The command `serve` starts an http server which provides a [REST API](../reference/search-api.md).
-Run it on each of your instances.
+Now ssh into the first EC2 instance, install Quickwit, and [configure the environment](configure-aws-env.md) to let Quickwit access the index S3 buckets.
 
 ```bash
-quickwit serve --index-uri s3://path-to-your-bucket/hdfs_logs --peer-seed=ip1,ip2,ip3
+curl -L https://install.quickwit.io | sh
+cd quickwit-v*/
 ```
 
-You will see in your terminal the confirmation that the instance has joined the cluster. Example of such a log:
+Let's create the node configuration file by first running this snippet. Make sure you fill in the correct values before running in the EC2 terminal session. 
 
+```bash
+export S3_PATH=s3://{path/to/bucket}/indexes
+export IP_NODE_1={first-ec2-instance-public-ip}
+export IP_NODE_2={second-ec2-instance-public-ip}
+export IP_NODE_3={third-ec2-instance-public-ip}
 ```
-INFO quickwit_cluster::cluster: Joined. host_key=019e4c0e-165a-430d-8ef6-7b7a035decac remote_host=Some(172.31.66.143:8081)
+
+```bash
+# configuration for our first node
+echo "version: 0
+node_id: searcher-1
+metastore_uri: ${S3_PATH}
+default_index_root_uri: ${S3_PATH}
+searcher:
+  rest_listen_address: 0.0.0.0
+  rest_listen_port: 7100
+  peer_seeds:
+    - ${IP_NODE_2}:7200 # searcher-2
+    - ${IP_NODE_3}:7300 # searcher-3
+" > config.yaml
 ```
 
-:::note
+Now let's launch a searcher node for this instance. 
 
-Quickwit by default, opens the 8080 port; it also needs the TCP and UDP 8081 port (8080+1) for cluster formation and
-finally, 8082 (8080+2) for gRPC communication between instances.
+```bash
+# Then start the http server search service.
+./quickwit service run searcher --config ./config.yaml
+```
 
-In AWS, you can create a security group to group these inbound rules. Check out the [network section](configure-aws-env.md) of our AWS setup guide.
+You will see in the terminal the confirmation that the instance has created a new cluster. Example of such a log:
 
-:::
+```
+INFO quickwit_cluster::cluster: Create new cluster. node_id="searcher-1" listen_addr=0.0.0.1:7200
+```
+
+Let's launch the second and third searcher nodes instance by repeating the same previous steps but each time with the respective configuration snippet.
 
+```bash
+# configuration for our second node
+echo "version: 0
+node_id: searcher-2
+metastore_uri: ${S3_PATH}
+default_index_root_uri: ${S3_PATH}
+searcher:
+  rest_listen_address: 0.0.0.0
+  rest_listen_port: 7200
+  peer_seeds:
+    - ${IP_NODE_1}:7100 # searcher-1
```

Comment:
if you want to simplify the tutorial, you don't need to declare all peer_seeds. Declaring peer seeds on node 1 is sufficient.

Reply:
And ideally, do not change `rest_listen_port`